services/web/server/src/simcore_service_webserver/director_v2_core_utils.py

```diff
@@ -0,0 +1,72 @@
+""" Misc that does not fit on other modules
+
+Functions/classes that are too small to create a new module or helpers within this plugin's context are placed here
+
+"""
+
+import asyncio
+import logging
+
+import aiohttp
+from aiohttp import ClientTimeout, web
+from models_library.projects import ProjectID
+
+from .director_v2_abc import AbstractProjectRunPolicy
+from .director_v2_settings import (
+    DirectorV2Settings,
+    get_client_session,
+    get_plugin_settings,
+)
+
+log = logging.getLogger(__name__)
+
+
+SERVICE_HEALTH_CHECK_TIMEOUT = ClientTimeout(total=2, connect=1)  # type:ignore
```

Comment:
maybe make this configureable as env-var/setting

Reply:
pair reviewed